certif/app/services/current-user.js

```diff
@@ -4,22 +4,18 @@ import { tracked } from '@glimmer/tracking';
 export default class CurrentUserService extends Service {
   @service session;
   @service store;
-  @tracked certificationPointOfContact;
-
-  get isFromSco() {
-    return this.certificationPointOfContact.isSco;
-  }
 
-  get currentCertificationCenter() {
-    return this.certificationPointOfContact.certificationCenters.findBy('id', String(this.certificationPointOfContact.currentCertificationCenterId));
-  }
+  @tracked certificationPointOfContact;
+  @tracked currentAllowedCertificationCenterAccess;
 
   async load() {
     if (this.session.isAuthenticated) {
       try {
-        this.certificationPointOfContact = await this.store.findRecord('certification-point-of-contact', this.session.data.authenticated.user_id, { include: 'certificationCenters' });
+        this.certificationPointOfContact = await this.store.findRecord('certification-point-of-contact', this.session.data.authenticated.user_id);
+        this.currentAllowedCertificationCenterAccess = this.certificationPointOfContact.hasMany('allowedCertificationCenterAccesses').value().objectAt(0);
```

Comment:
La syntaxe: `this.certificationPointOfContact.allowedCertificationCenterAccesses.firstObjet;` ne fonctionne pas ? Après la tienne se comprend très bien aussi.

Reply:
`this.certificationPointOfContact.allowedCertificationCenterAccesses.value().firstObjet`